unison-share-api/src/Unison/Server/Backend.hs

```diff
@@ -633,12 +636,17 @@ fixupNamesRelative root = Names.map fixName
 --
 -- Construct a 'Search' with 'makeTypeSearch' or 'makeTermSearch', and eliminate it with 'applySearch'.
 data Search r = Search
-  { lookupNames :: r -> Set (HQ'.HashQualified Name),
-    lookupRelativeHQRefs' :: HQ'.HashQualified Name -> Set r,
+  { lookupNames :: r -> (Set (HQ'.HashQualified Name)),
```

Comment:
Why more parens? (Not necessarily worth removing if accidental, just wondering if it wasn't)

Reply:
I think I had embedded this behind a monad at one point when working on this, then must've missed the parens when I removed the monad again, I can undo the diff 👍🏼